mtags/src/main/scala/scala/meta/internal/mtags/OnDemandSymbolIndex.scala

```diff
@@ -5,7 +5,7 @@ import java.nio.charset.StandardCharsets
 import scala.collection.concurrent.TrieMap
 import scala.meta.inputs.Input
 import scala.meta.internal.io.PathIO
-import scala.meta.internal.io._
+import scala.meta.internal.io.{ListFiles => _, _}
```

Comment:
Why do we do `ListFiles => _` here? I don't see any change conencted to that.

Reply:
Great question, I don't know 😅
On Scala 2.13 the `ListFiles` from `scala.meta.internal.io` takes precedence over our `ListFiles` so it doesn't compile.

Reply:
Ach alright, I didn't know we had multiples of that. Thanks!